disnake/interactions/base.py

```diff
@@ -188,6 +189,7 @@ def __init__(self, *, data: InteractionPayload, state: ConnectionState):
         self.token: str = data["token"]
         self.version: int = data["version"]
         self.application_id: int = int(data["application_id"])
+        self._app_permissions: int = int(data.get("app_permissions", 0))
```

Comment:
Should be added to `disnake.types.interactions` as well

Reply:
https://github.com/DisnakeDev/disnake/pull/586/commits/5771815acbf32d25b528833cee31ffeaf05d7987